node/pkg/watchers/evm/reobserve.go

```diff
@@ -68,7 +74,13 @@ func (w *Watcher) handleReobservationRequest(ctx context.Context, chainId vaa.Ch
 					zap.Uint64("current_safe_block", safeBlockNum),
 					zap.Uint64("observed_block", blockNumber),
 				)
-				w.msgC <- msg
+
+				pubErr := w.verifyAndPublish(msg, ctx, eth_common.BytesToHash(msg.TxID), receipt)
+
+				if pubErr != nil {
+					w.logger.Error("Error when publishing message", zap.Error(err))
```

Comment:
You should continue so `numObservations` is not pegged.

Reply:
Should be done in the latest commit.